R/zchunk_LB172.nonghg_elc_USA.R

```diff
@@ -0,0 +1,165 @@
+#' module_gcamusa_LB172.nonghg_elc
+#'
+#' Base-year and future year electricity non-ghg  emissions by U.S. state / fuel / pollutant / year
+#'
+#' @param command API command to execute
+#' @param ... other optional parameters, depending on command
+#' @return Depends on \code{command}: either a vector of required inputs,
+#' a vector of output names, or (if \code{command} is "MAKE") all
+#' the generated outputs: \code{L172.nonghg_tg_state_elec_F_Yb}, \code{L172.nonghg_tgej_state_elec_F_Yf}. The corresponding file in the
+#' original data system was \code{LB172.nonghg_elc_USA.R} (gcam-usa level1).
+#' @details elec sector non-ghg emissions by U.S. state / sector / fuel / pollutant / base and future year.
```

Comment:
This isn't much of a `@details` section--can you expand and clarify this please? What exactly are the steps that occur?